patches/@stripe+stripe-react-native+0.45.0.patch

```diff
@@ -0,0 +1,6164 @@
+diff --git a/node_modules/@stripe/stripe-react-native/android/build/.transforms/07acbda58c5b3a635889c8ad58af3151/results.bin b/node_modules/@stripe/stripe-react-native/android/build/.transforms/07acbda58c5b3a635889c8ad58af3151/results.bin
+new file mode 100644
+index 0000000..0d259dd
+--- /dev/null
++++ b/node_modules/@stripe/stripe-react-native/android/build/.transforms/07acbda58c5b3a635889c8ad58af3151/results.bin
+@@ -0,0 +1 @@
++o/classes
+diff --git a/node_modules/@stripe/stripe-react-native/android/build/.transforms/07acbda58c5b3a635889c8ad58af3151/transformed/classes/classes_dex/classes.dex b/node_modules/@stripe/stripe-react-native/android/build/.transforms/07acbda58c5b3a635889c8ad58af3151/transformed/classes/classes_dex/classes.dex
+new file mode 100644
+index 0000000..072af7e
+Binary files /dev/null and b/node_modules/@stripe/stripe-react-native/android/build/.transforms/07acbda58c5b3a635889c8ad58af3151/transformed/classes/classes_dex/classes.dex differ
+diff --git a/node_modules/@stripe/stripe-react-native/android/build/.transforms/ea2e5d6ca89d59388b921dd6c1ad61b5/results.bin b/node_modules/@stripe/stripe-react-native/android/build/.transforms/ea2e5d6ca89d59388b921dd6c1ad61b5/results.bin
+new file mode 100644
+index 0000000..7ed749e
+--- /dev/null
++++ b/node_modules/@stripe/stripe-react-native/android/build/.transforms/ea2e5d6ca89d59388b921dd6c1ad61b5/results.bin
+@@ -0,0 +1 @@
++o/bundleLibRuntimeToDirDebug
+diff --git a/node_modules/@stripe/stripe-react-native/android/build/.transforms/ea2e5d6ca89d59388b921dd6c1ad61b5/transformed/bundleLibRuntimeToDirDebug/bundleLibRuntimeToDirDebug_dex/PaymentOptionDisplayDataMapperKt.dex b/node_modules/@stripe/stripe-react-native/android/build/.transforms/ea2e5d6ca89d59388b921dd6c1ad61b5/transformed/bundleLibRuntimeToDirDebug/bundleLibRuntimeToDirDebug_dex/PaymentOptionDisplayDataMapperKt.dex
```

Comment:
All of the patches contain information about build data, which is incorrect.

Reply:
Should be fixed in a previous PR on the stack (Graphite).

Reply:
Rebase problem, sorry